apps/odo/config/integration.exs

```diff
@@ -0,0 +1,64 @@
+use Mix.Config
+# We should turn this into a repo to share with Reaper
```

Comment:
^ Do we get this for free now that they're all umbrella?

Reply:
yes but that block should theoretically be able to be removed now that we don't have any integration tests for Odo (they should all be covered by unit tests or e2e).